source/MRViewer/MRObjectTransformWidget.h

```diff
@@ -129,6 +149,7 @@ class MRVIEWER_CLASS ObjectTransformWidget : public MultiListener<MouseDownListe
     bool picked_{ false };
     bool pickThrough_{ false };
 
+    std::function<void( float )> scaleTooltipCallback_;
```

Comment:
i think we need to reset it in create or reset function (like other callbacks)

Reply:
Fixed.